app/views/plans/_overview_details.html.erb

```diff
@@ -32,7 +32,8 @@
                   <div class="pull-left">
                     <%= _("%{phase_title} (%{sections_size} %{sections}, %{questions_size} %{questions})") % {
                       :phase_title => p.title, :sections_size => p.sections.size, :sections => n_('section', 'sections', p.sections.size),
-                      :questions_size => p.questions.size, :questions => n_('question', 'questions', p.questions.size) } %>
+                      :questions_size => p.sections.map{|s| s.questions.size }.inject(0){|sum,x| sum + x }, 
```

Comment:
I'd make two changes:
- Extract the behaviour ```p.sections.map...``` into a variable
- use reduce instead of inject, although they are aliases, we are using reduce across the service and keeps the code more consistent.